src/Microsoft.Graph.Core/Requests/Content/BatchResponseContent.cs

```diff
@@ -39,6 +45,7 @@ public BatchResponseContent(HttpResponseMessage httpResponseMessage, ISerializer
             });
 
             this.Serializer = serializer ?? new Serializer();
+            this.ResponseHandler = new ResponseHandler(this.Serializer);
```

Comment:
shouldn't you also check the parameter of the ctor?

Reply:
Sorry about this. This was my mistake. 🤦 
This should be resolved in the latest update.